src/power_grid_model/core/power_grid_model.py

```diff
@@ -353,55 +360,54 @@ def calculate_state_estimation(
         continue_on_batch_error: bool = False,
     ) -> Dict[str, np.ndarray]:
         """
-        Calculate state estimation once with the current model attributes.
-        Or calculate in batch with the given update dataset in batch
+        Calculate state estimation once with the current model attributes. 
```

Comment:
```suggestion
        Calculate state estimation once with the current model attributes.
```

Reply:
The original docstring in the main branch contains two lines.

"Calculate state estimation once with the current model attributes.
  Or calculate in batch with the given update dataset in batch"

Reply:
still, I don't think there should be any trailing whitespaces in the code ever. I think the converter should transform enters into whitespaces, no?